reporting/lsrd_stats.py

```diff
@@ -1,675 +1,441 @@
+
+    with DBConnect(**dbinfo) as db:
+        db.select(sql, (begin_date, end_date))
+        for i, key in enumerate(sorted(infodict.keys())):
+            infodict[key] = int(db[0][i])
+
+    return infodict
+
+
+def calc_dlinfo(log_file):
+    """
+    Count the total tarballs downloaded and their combined size
+
+    :param log_file: Combined Log Format file path
+    :type log_file: str
+    :return: Dictionary of values
+    """
+    infodict = {'tot_dl': 0,
+                'tot_vol': 0.0}
+
+    # (ip, logname, user, datetime, method, resource, status, size, referrer, agent)
+    regex = r'(.*?) (.*?) (.*?) \[(.*?)\] "(.*?) (.*?) (.*?)" (\d+) (\d+) "(.*?)" "(.*?)"'
+
+    with open(log_file, 'r') as log:
+        for line in log:
+            try:
+                gr = re.match(regex, line).groups()
+                # Kept simple for ease of reading and future use
+                if gr[7] == '200' and gr[4] == 'GET' and '.tar.gz' in gr[5] and '/orders/' in gr[5]:
+                    infodict['tot_vol'] += int(gr[8])
+                    infodict['tot_dl'] += 1
+            except:
+                continue
+
+    # Bytes to GB
+    infodict['tot_vol'] /= 1073741824.0
+
+    return infodict
+
+
+def db_scenestats(source, begin_date, end_date, dbinfo):
+    """
+    Queries the database for the number of scenes ordered
+    separated by USGS and non-USGS emails
+    dates are given as ISO 8601 'YYYY-MM-DD'
+
+    :param source: EE or ESPA
+    :type source: str
+    :param begin_date: Date to start the count on
+    :type begin_date: str
+    :param end_date: Date to stop the count on
+    :type end_date: str
+    :param dbinfo: Database connection information
+    :type dbinfo: dict
+    :return: Dictionary of the counts
+    """
+    sql = ('''select COUNT(*)
+              from ordering_scene
+              inner join ordering_order on ordering_scene.order_id = ordering_order.id
+              where ordering_order.order_date::date >= %s
+              and ordering_order.order_date::date <= %s
+              and ordering_order.orderid like '%%@usgs.gov-%%'
+              and ordering_order.order_source = %s;''',
+           '''select COUNT(*)
+              from ordering_scene
+              inner join ordering_order on ordering_scene.order_id = ordering_order.id
+              where ordering_order.order_date::date >= %s
+              and ordering_order.order_date::date <= %s
+              and ordering_order.orderid not like '%%@usgs.gov-%%'
+              and ordering_order.order_source = %s;''')
+
+    results = {'sc_month': 0,
+               'sc_usgs': 0,
+               'sc_non': 0}
+
+    with DBConnect(**dbinfo) as db:
+        for q in sql:
+            db.select(q, (begin_date, end_date, source))
+
+            if 'not like' in q:
+                results['sc_non'] += int(db[0][0])
+            else:
+                results['sc_usgs'] += int(db[0][0])
+
+    results['sc_month'] = results['sc_usgs'] + results['sc_non']
```

Comment:
its better to take up more space when naming vars so it's evident what they are... results['scenecount_month'] rather than results['sc_month'].  Or maybe counts['month'], counts['usgs']?